prow/plank/controller.go

```diff
@@ -98,6 +103,16 @@ type Controller struct {
 
 	// if skip report job results to github
 	skipReport bool
+
+	//prowJob Client
+	//prowJobClient prowv1.ProwJobInterface
+
+	// pod client
+	podClients []corev1.PodInterface
```

Comment:
These clients are already available as `buildClients` on line 83

Reply:
This comment still applies, please remove the duplicate clients here and use the `.buildClients` property from the `Controller`